src/shared-components/typography/index.ts

```diff
@@ -148,14 +155,3 @@ export const Typography = {
   SuccessText: Success,
```

Comment:
how about removing these exported members here as well?

Reply:
Missed that, thanks! Fixed in 0e8e2fb8714507944dd6ed026ac8eb69861d2f47